internal/region/region.go

```diff
@@ -0,0 +1,62 @@
+package region
+
+// Region represents the sauce labs region.
+type Region uint
+
+const (
+	// None is an undefined sauce labs region.
+	None Region = iota
+	// USWest1 is a sauce labs region in the US, aka us-west-1.
+	USWest1
+	// EUCentral1 is a sauce labs region in the EU, aka eu-central-1.
+	EUCentral1
+	// Staging is an internal sauce labs region.
+	Staging
+)
+
+var meta = []struct {
+	Name       string
+	APIBaseURL string
+}{
+	// None
+	{
+		"",
+		"",
+	},
+	// USWest1
+	{
+		"us-west-1",
+		"https://api.us-west-1.saucelabs.com",
+	},
+	// EUCentral1
+	{
+		"eu-central-1",
+		"https://api.eu-central-1.saucelabs.com",
+	},
+	// Staging
+	{
+		"staging",
+		"https://api.staging.saucelabs.net",
+	},
```

Comment:
@alexplischke what do you think about adding another flag for specifying the API endpoint? like `-x` like we use in sauce connect or sauce-runner?